scenarioo-client/protractor-e2e.conf.js

```diff
@@ -29,13 +29,14 @@ var exportsConfig = {
     allScriptsTimeout: 20000,
     getPageTimeout: 20000,
 
-    specs: [/* See gulpfile.js for specified tests */],
+    specs: ['./test/protractorE2E/specs/**/*.js'],
 
     capabilities: {
         browserName: 'chrome',
         chromeOptions: {
             args : [
-                'disable-infobars'
+                'disable-infobars',
+                'window-size=1280,800',
```

Comment:
This change is also needed in the other protractor conf for e2e tests with scenarioo reports!
We should once introduce a common base config that is reused for both configs to avoid such mistakes.